easybuild/easyblocks/g/gcc.py

```diff
@@ -84,6 +86,11 @@ def extra_options():
             'clooguseisl': [False, "Use ISL with CLooG or not", CUSTOM],
             'generic': [None, "Build GCC and support libraries such that it runs on all processors of the target "
                               "architecture (use False to enforce non-generic regardless of configuration)", CUSTOM],
+            'rename_include-fixed': [False, "Rename the 'include-fixed such that it is not used when using GCC. "
```

Comment:
This has to be `rename_include_fixed`, you can't have `-` in variable names in Python (and hence same for easyconfig parameters)

```suggestion
            'rename_include_fixed': [False, "Rename the 'include-fixed' directory to avoid that it is used by GCC. "